### ISSUE TITLE
how can I make the mod work with puredark's RDR2 FG mod?

### ISSUE THREAD

**Issue by AmSoFancy** [Nukem9/dlssg-to-fsr3#220]
is there any way to make the mod work with Puredark's RDR2 mod? I tried every way but it keeps crashing after the story loads. I tried it with Elden Ring and it worked.

**Comment by TimmyP777** [comment]
I got it to work but it seems Im the only one. Im using a cracked exe to avoid the launcher. Try it.

**Comment by Vairy02** [comment]
> I got it to work but it seems Im the only one. Im using a cracked exe to avoid the launcher. Try it.

hi, how you got it to work with rdr2, i have puredark mod and put dlssg to fsr3 mod in the game folder and mods folder, but when i enable dlssg on the rdr2 menu the game goes total black

**Comment by TimmyP777** [comment]
I guess it might be the cracked exe to avoid using the launcher. Otherwise I dont know why it works, but it does... wonderfully. So get ready cause you will get it eventually.

**Comment by AmSoFancy** [comment]
> I guess it might be the cracked exe to avoid using the launcher. Otherwise I dont know why it works, but it does... wonderfully. So get ready cause you will get it eventually.

so I just get the legit copy of the game and apply the cradck over it?

**Comment by philmatik** [comment]
> I got it to work but it seems Im the only one. Im using a cracked exe to avoid the launcher. Try it.

Did you use the dlss tweaks version or no?

**Comment by philmatik** [comment]
@TimmyP777 did you use the dlss tweaks version or no?

**Comment by TimmyP777** [comment]
Yeah. I think the crack is for an older version of the game, but it works on newer releases.

Just remember, I might not have the newest release but just a newer release.

*No DLSS tweak version. Havent had to use it yet for anything.

**Comment by AmSoFancy** [comment]
is there any chance you could tell me your version of the game please? you can see it on the bottom right in the game menu, thanks

**Comment by philmatik** [comment]
Not working for me with .50, .60, or .70 versions with 1436.28 build. I'm not using the rockstar launcher either. Using .70 I get the black screen, with .50 and .60 i get a popup from the game on crash.

**Comment by TimmyP777** [comment]
Here I made this to show in game how the setup looks. Dont pay attention to the fps counter, its not correct and goes down when I enable FG, but performance obviously goes up. My keyboard fps monitor says about 65fps to 110fps...

Anyways. DX12, vsync off triple buffering off using exe crack. Vsync forced in nvcp, also changed vsync from adaptive to standard (defaulting put it at adaptive, dont know if it did anything). Rebar on.

https://www.youtube.com/watch?v=Qv9pKgwRZ34

**Comment by AmSoFancy** [comment]
> Here I made this to show in game how the setup looks. Dont pay attention to the fps counter, its not correct and goes down when I enable FG, but performance obviously goes up. My keyboard fps monitor says about 65fps to 110fps...
> 
> Anyways. DX12, vsync off triple buffering off using exe crack. Vsync forced in nvcp, also changed vsync from adaptive to standard (defaulting put it at adaptive, dont know if it did anything). Rebar on.
> 
> https://www.youtube.com/watch?v=Qv9pKgwRZ34
> 
> *Remember this video is just to show settings and the log, its not to show performance improvements and I dont think they are gonna come through on YT, not even sure I recorded this at 60fps. They gains are massive though.

thank you for showing us, Ill try a different version now

**Comment by philmatik** [comment]
@TimmyP777 What rdr2 build are you using? It's bottom right when you first launch the game.

**Comment by TimmyP777** [comment]
1436.28 I figured that was just the build of the cracked exe. So the crack always says 1436.28.

However now that I think about it. I recently archived all my installed games to an HDD, and ones that were large in size... I re-downloaded elsewhere if they saved 80gb, in some cases. This might have been one of them. So that might mean...

**Comment by philmatik** [comment]
@TimmyP777 All good, thanks for all the info.

**Comment by TimmyP777** [comment]
No prob. Post if you get it!

**Comment by MinaAlmighty** [comment]
whenever I try to change the graphical settings the game crashes, any idea how to fix it ?

**Comment by Vairy02** [comment]
> whenever I try to change the graphical settings the game crashes, any idea how to fix it ?

Remove d3d12.dll from the game folder, enter the game, change the settings you need to change, close the game, and put d3d12.dll in the game folder

**Comment by MinaAlmighty** [comment]
> > whenever I try to change the graphical settings the game crashes, any idea how to fix it ?
> 
> Remove d3d12.dll from the game folder, enter the game, change the settings you need to change, close the game, and put d3d12.dll in the game folder

When I do this the DLSS option disappears which is the main setting I wanted to change..

**Comment by TimmyP777** [comment]
\Documents\Rockstar Games\Red Dead Redemption 2\Settings\system.xml

**Comment by MinaAlmighty** [comment]
> \Documents\Rockstar Games\Red Dead Redemption 2\Settings\system.xml

Thank you, I changed all the settings except the dlss. I can't change it from performance preset. How can I change it ?

**Comment by MinaAlmighty** [comment]
> \Documents\Rockstar Games\Red Dead Redemption 2\Settings\system.xml

Thank you

**Comment by TimmyP777** [comment]
You got DLSS changes you wanted?

**Comment by MinaAlmighty** [comment]
> es you wanted?

Yeah. I kept playing around with them till I got it. Thanks a lot

**Comment by Vairy02** [comment]
now it work with 0.80, i used the non-dlsstweak version, 0.70 was giving black screen after enabling FG, now seems to work correctly, cracked rdr2 1436.28

**Comment by AmSoFancy** [comment]
yea I finally got it to work with 0.80, I'm using the DLSS tweaks version so I can force DLAA, the HUD flicker is annoying though

**Comment by Styphoryte** [comment]
> yea I finally got it to work with 0.80, I'm using the DLSS tweaks version so I can force DLAA, the HUD flicker is annoying though

So how exactly did you guys get this to work in RDR2 combined with PureDark right? Did you change any of the files in the Puredark "Mods" folder in the "Streamline" folder you didn't need to replace nvngx_dlssg.dll at all? Just copy over PureDark and dlssg to fsr3? My problem is when I try this, using same build 1436.28 cracked it just opens to a black screen with my cursor spinning around until it eventually closes by itself. I think it's because I'm trying to run my game in Vulkan mode, maybe that's the reason? If so, I guess I'll hold off on it for RDR2 since mostly all my mods require Vulkan runtime anyhow.

**Comment by Vairy02** [comment]
> > yea I finally got it to work with 0.80, I'm using the DLSS tweaks version so I can force DLAA, the HUD flicker is annoying though
> 
> So how exactly did you guys get this to work in RDR2 combined with PureDark right? Did you change any of the files in the Puredark "Mods" folder in the "Streamline" folder you didn't need to replace nvngx_dlssg.dll at all? Just copy over PureDark and dlssg to fsr3? My problem is when I try this, using same build 1436.28 cracked it just opens to a black screen with my cursor spinning around until it eventually closes by itself. I think it's because I'm trying to run my game in Vulkan mode, maybe that's the reason? If so, I guess I'll hold off on it for RDR2 since mostly all my mods require Vulkan runtime anyhow.

Probably its because you are using Vulkan, these mod only work with DirectX12

**Comment by trxsudo** [comment]
> is there any way to make the mod work with Puredark's RDR2 mod? I tried every way but it keeps crashing after the story loads. I tried it with Elden Ring and it worked.

could you share how you get elden ring frame gen working?

**Comment by Styphoryte** [comment]
> > > yea I finally got it to work with 0.80, I'm using the DLSS tweaks version so I can force DLAA, the HUD flicker is annoying though
> > 
> > 
> > So how exactly did you guys get this to work in RDR2 combined with PureDark right? Did you change any of the files in the Puredark "Mods" folder in the "Streamline" folder you didn't need to replace nvngx_dlssg.dll at all? Just copy over PureDark and dlssg to fsr3? My problem is when I try this, using same build 1436.28 cracked it just opens to a black screen with my cursor spinning around until it eventually closes by itself. I think it's because I'm trying to run my game in Vulkan mode, maybe that's the reason? If so, I guess I'll hold off on it for RDR2 since mostly all my mods require Vulkan runtime anyhow.
> 
> Probably its because you are using Vulkan, these mod only work with DirectX12

Hopefully one day.... But who knows maybe FSR3 will only work with D3D12 games for now.

**Comment by Capoos** [comment]
Is the latest version of Puredark's RDR2 mod Beta Build 02?
